project/pd/project.json

```
@@ -0,0 +1,36 @@
+{
+    "comments": [
```

```suggestion
    "_comment": [
```
Why do we need to keep the letter sequence? It doesn't seem to matter if we don't keep it?

I think if we don't have a tool to check it, then it's hard to make sure it stays alphabetical. I would actually prefer to remove the comment and use a tool to make sure that the changes are alphabetical. Maybe it would be better to use eslint or a prettier plugin to check in CI. If you don't have time to add it, then I suggest we remove the comments and open an issue to track it and add it later.

---

> Why do we need to keep the letter sequence?

Better to read in general. I don't think it is a strong requirement but we can setup a rule so that others can also propose a fix if it is out of order. See also TiDB document list variables in order and we don't have a linter. I will be glad if there is a linter and agree to open an issue. But let's keep the comment.

---

Add a comment https://github.com/tikv/community/issues/121#issuecomment-858235137, I don't think we should create an issue before this pull request merged. It is invalid until then.

---

ping @hi-rustin

---

OK. Got it. Thanks!